### PR TITLE
Fixed: JENKINS-40925 - dir context is not honored by shell step

### PR DESCRIPTION
**Pull request by electroma**
It looks too easy. 
Please let me know if there is a reason not to use command's pwd.

### CODE REVIEW

**Comment by iocanel** [comment]
I am not familiar with dir context, but the code looks good and tests seem to pass.
@carlossg can you have a look too?

**Comment by electroma** [comment]
@carlossg can you have a look?

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java [hunk 1]
```diff
@@ -72,7 +70,6 @@ public ContainerExecDecorator(KubernetesClient client, String podName, String co
         this.client = client;
         this.podName = podName;
         this.containerName = containerName;
-        this.path = path;
```
Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java]: if path is removed, then the constructor should be deprecated and a new one without path argument added

**Comment by electroma** [comment]
@carlossg there is only one usage of this constructor, so I can just change the signature inplace. 
Do I still need to deprecate and create new one?

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java (hunk 1 above)
Comment (electroma) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java]: @carlossg there is only one usage of this constructor, so I can just change the signature inplace.
Do I still need to deprecate and create new one?

Comment (electroma) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java]: I've removed unused parameter

Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java]: yes, better to copy and deprecate to keep backwards binary compatibility

Comment (electroma) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java]: done

Comment (electroma) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java]: @carlossg green light

**Pull request merged**